tests/pytest_runner.py

```diff
@@ -1,17 +1,29 @@
+import glob
 import os
 
 import pytest
 
-from yt.config import ytcfg
-
 if __name__ == "__main__":
     os.environ["OMP_NUM_THREADS"] = "1"
     pytest_args = [
-        f"--local-dir={os.path.join(ytcfg.get('yt', 'test_data_dir'), 'answers')}",
-        "-c=pytest_answer.ini",
-        "--junitxml=unittests.xml",
-        "--answer-big-data",
```

Comment:
Do we need the `big-data` thing at all anymore?

Reply:
Nope. We just need to make sure it won't sneak in with subsequent PRs.

Reply:
Ok! Is it not relevant to any of our testing anymore (or is the decorator still relevant)? We should make sure we get rid of references in the docs too (maybe PR 3 also?).

Reply:
You mean a doc PR like this one: https://github.com/yt-project/yt/pull/3131 ? ;)

Reply:
You know it!!! 🙂